bugsnag-android-core/src/androidTest/java/com/bugsnag/android/DeviceDataSummaryTest.java

```diff
@@ -1,40 +0,0 @@
-package com.bugsnag.android;
-
-import static org.junit.Assert.assertNotNull;
-
-import android.content.Context;
-import android.content.res.Resources;
-
-import androidx.test.core.app.ApplicationProvider;
-
-import org.junit.Before;
-import org.junit.Test;
-
-import java.util.Map;
-
-public class DeviceDataSummaryTest {
-
-    private Map<String, Object> deviceData;
-
-    /**
-     * Generates a device data object
-     */
-    @Before
-    public void setUp() {
-        Connectivity connectivity = BugsnagTestUtils.generateConnectivity();
-        Context context = ApplicationProvider.getApplicationContext();
-        Resources resources = context.getResources();
-        DeviceData deviceData = new DeviceData(connectivity, context, resources,
-                "123", DeviceBuildInfo.Companion.defaultInfo());
-        this.deviceData = deviceData.getDeviceDataSummary();
-    }
-
-    @Test
-    public void testAccessors() {
-        assertNotNull(deviceData.get("manufacturer"));
-        assertNotNull(deviceData.get("model"));
-        assertNotNull(deviceData.get("osName"));
-        assertNotNull(deviceData.get("osVersion"));
-    }
```

Comment:
I was expecting to see these (and `DeviceDataTest`) tests moved to a unit test (as opposed to instrumentation) with assertions on each of the getters for each of the fields in DeviceData.

Reply:
`deviceData` in this test is a `Map`, so adding similar assertions would just be testing `Map#get()`. I have therefore avoiding doing this as we already check the values of each field by verifying the JSON matches the test fixture.